packaging/rpm/rear.spec

```diff
@@ -193,6 +193,7 @@ OS_VERSION="13.2"
 %doc %{_mandir}/man8/rear.8*
 %config(noreplace) %{_sysconfdir}/cron.d/rear
 %config(noreplace) %{_sysconfdir}/rear/
+%config(noreplace) %{_sysconfdir}/rear/cert/
```

Comment:
@didacog was that necessary? It seems that `%config(noreplace) %{_sysconfdir}/rear/` includes also `%config(noreplace) %{_sysconfdir}/rear/cert/` and the build process complains:   `warning: File listed twice: /etc/rear/cert`.
Unfortunately, I have not found any docs on what is `%config(noreplace)` supposed to do when given a directory, so I had to experiment. It seems that it marks the files under the directory as config, but the directory itself can't be marked as config, and since the directory is emtpy, it would not have any effect anyway.